docs/docs/user_management/user_roles.md

```diff
@@ -1,3 +1,10 @@
 # User Roles in Minder
 
-Explains user roles
+When incorporating a user into your project, it's crucial to assign them the appropriate role based on their responsibilities and required access levels.
+Minder currently offers the following roles:
+
+- `viewer`: Provides read-only access to the project. Users with this role can view associated resources such as enrolled repositories, rule types, and profiles.
+- `editor`: Grants the same permissions as the viewer role, along with the ability to edit project resources, excluding the project itself and the list of providers.
+- `admin`: Grants administrator rights on the project. Users with this role have the same permissions as editor and can also modify the project and associated providers.
```

Comment:
Do you want to say "full permissions" here?  Maybe also mention that administrators can add and remove users from other roles?